src/Disks/IDisk.cpp

```diff
@@ -7,9 +7,6 @@
 #include <Common/logger_useful.h>
 #include <Common/setThreadName.h>
 #include <Core/ServerUUID.h>
-#include <Disks/ObjectStorages/MetadataStorageFromDisk.h>
-#include <Disks/ObjectStorages/FakeMetadataStorageFromDisk.h>
```

Comment:
Maybe we can remove whole `FakeMetadataStorageFromDisk` now?

Reply:
Indeed!